controllers/config/config_test.go

```diff
@@ -16,13 +15,13 @@ func TestNewOperatorConfig(t *testing.T) {
 	_ = os.Unsetenv("DOCKER_REGISTRY")
 
 	expectedNamespace := "myNamepsace"
-	expectedDoguRegistryData := config.DoguRegistryData{
+	expectedDoguRegistryData := DoguRegistryData{
 		Endpoint: "myEndpoint",
 		Username: "myUsername",
 		Password: "myPassword",
 	}
 	inputDockerRegistrySecretData := "{\"auths\":{\"your.private.registry.example.com\":{\"username\":\"myDockerUsername\",\"password\":\"myDockerPassword\",\"email\":\"jdoe@example.com\",\"auth\":\"c3R...zE2\"}}}"
```

Comment:
Tests are a sort of documentation. Triple backticks ` ``` "key"="value" ``` `can drastically reduce the mental load of reading this JSON text. I often find myself looking into tests to scout for examples or to grasp finer details of configuration.

Reply:
I think you meant the single backticks 
``` 
`"key"="value"`
```
Done.